### ISSUE TITLE
i18n

### ISSUE THREAD

**Issue by yurtpage** [quillpad/quillpad#305]
To make it easier to translate there exists a few translation services:

* https://hosted.weblate.org/ from creator of the Weblate online platform that itself is FOSS. It has pre-moderation for FOSS projects. It doesn't offer free automatic translations, only the LibreTranslate that almost not working. But you can buy and integrate own API key to Google Translate, Yandex, DeepL, Baidu, etc. It will periodically send translations as a PR on the GitHub.
* https://translate.codeberg.org/ also based on Weblate, doesn't have a pre-moderation which is a plus. It can't send a PR on GitHub because they are competitors.
* https://toolate.othing.xyz/ based on Weblate, no pre-moderation, can send a PR, has automatic translations from Baidu that gives good results. It's supported by F-Droid translators and hosted in Singapore.
* Crowdin.com has pre-moderation for FOSS, has a better UI.
* Transifex, Lokalize.com, PoEditor.com maybe some other may be also good but less often used.

**Comment by jsixface** [comment]
Can I trust the translations? I don't know any of these languages. Are these done by humans or auto-translated?

I would like a PR for the README if this is done by volunteers.

**Comment by yurtpage** [comment]
Yes, you can _generally_ trust to the translations. They are made by humans but the Weblate has a feature of automatic translation that shows a translation variant. A human translator needs to check it and modify if needed but it's always a human submits a translation.
There is an ability to add voting or reviewers but this almost newer needed. 
Given that almost all strings for most popular languages are already translated this is safe to use. Translators will be notified when a new string is added so that they can add a translation immediately.

**Comment by reysonk** [comment]
The Russian translation is correct
